pkg/cloudprovider/aws/apis/v1alpha1/provider_defaults.go

```diff
@@ -65,3 +69,35 @@ func (c *Constraints) defaultAMIFamily() {
 	}
 	c.AMIFamily = &AMIFamilyAL2
 }
+
+func (c *Constraints) defaultBlockDeviceMappings() {
```

Comment:
After discussing w/ controller-runtime. I think we should do in-memory default instead of API level defaulting.